src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyContentService.cs

```diff
@@ -458,42 +464,44 @@ private List<ContentSectionViewModel> OrderedContentSections(List<ContentSection
             return tuple;
         }
 
-        private Either<ActionResult, Tuple<MethodologyVersionContent, List<ContentSection>>> FindContentList(
-            MethodologyVersionContent content,
-            params Guid[] contentSectionIds)
+        private static List<ContentSection> GetContentListWithSection(MethodologyVersionContent content,
+            ContentSection contentSection)
         {
-            return FindContentList(content, contentSectionIds.ToList());
+            return FindContentListWithSectionId(content, contentSection.Id).Right.Item2;
```

Comment:
I think you can do something like this and then make this line more readable than having `Item2`:
```
        private static Either<ActionResult, (MethodologyVersionContent Content, List<ContentSection> ContentSectionList)>
            FindContentListWithSectionId(MethodologyVersionContent content,
                Guid contentSectionId)
        {
            if (content.Content.Any(section => section.Id == contentSectionId))
            {
                return (Content: content, ContentSectionList: content.Content);
            }

            if (content.Annexes.Any(section => section.Id == contentSectionId))
            {
                return (Content: content, ContentSectionList: content.Annexes);
            }

            return new NotFoundResult();
        }
```

Reply:
I ended up rewriting `RemoveContentSection` to find the section to remove from the list again.

`var sectionToRemove = contentSections.Single(section => section.Id == contentSectionId);`

This isn't ideal because it's already in one of the other tuples, but I was then able to remove this extra method, and `FindContentListWithSectionId` remains consistent in its use of tuples with the other methods.

I think the use of tuples as method parameters and return types in this class is a bit crazy and it could be a lot simpler if we didn't use them. We'd probably just need to make more use of `OnSuccessCombineWith` to clean a lot of them up.